### PR TITLE
Feature 4: Completed

### LINKED ISSUE #4: Feature: Store
**Describe the solution you'd like**
As a user, I want my entries to be stored and persisted somewhere.

- [ ] Each entry should be stored.
- [ ] Each entry should be updatable from the stored file.

### PR DESCRIPTION
**Pull request by Linzeur**
I create methods to read, save and update data in a file.
Resolved #4

### CODE REVIEW

File: controller/entry_methods.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  data_file = File.read($path_data)
+  data_file = "{}" if data_file == "" 
+  data = JSON.parse(data_file)
+  data
+end
+
+def save_data(entry)
+  data_file = read_data
+  identifier = Time.now.getutc.to_i
+  data_file[identifier] = entry
```
Comment (sergiodxa) [inline controller/entry_methods.rb]: The entry doesn't have an ID key?

Comment (Linzeur) [reply controller/entry_methods.rb]: It has, the key of json file is the identifier

Comment (sergiodxa) [reply controller/entry_methods.rb]: But I mean inside the entry, like `{ "id": 123, "title": "a random title" }`

Comment (Linzeur) [reply controller/entry_methods.rb]: Not, it is like {"1234" : {"title": "titulo 1", "content":"contenido1"}}, "1234" is the id

Comment (sergiodxa) [reply controller/entry_methods.rb]: I recommend you to add the ID as a key to the entry itself too. That way you could have the whole entry including the ID in a single hash.

Comment (Linzeur) [reply controller/entry_methods.rb]: You're right, I will consider that

Comment (Linzeur) [reply controller/entry_methods.rb]: Sergio, in the line 15 I add the key id.

**Pull request merged**